tokens/src/font/color.json

```diff
@@ -3,7 +3,7 @@
     "color": {
       "heading": { "value": "{color.narmi.black.value}" },
       "primary": { "value": "{color.narmi.grey.value}" },
-      "secondary": { "value": "{color.narmi.mediumGrey.value}" },
+      "secondary": { "value": "#6c6c6c" },
```

Comment:
it's okay that this one is hardcoded but the others are variable?

Reply:
Yep, that's fine. The aliases point to hard coded values anyway. The `mediumGrey` token is still available, but secondary text is no longer pointing at at so we can give a slightly darker value to text here